daemon/logger/fluentd/fluentd.go

```diff
@@ -48,7 +48,11 @@ const (
 	defaultRetryWait  = 1000
 	defaultMaxRetries = math.MaxInt32
 
+	// Write() will not time out
+	defaultWriteTimeout = time.Duration(0)
```

Comment:
We should still set a default timeout. The main thing we don't want to happen is to allow a dead logging backend lock up the daemon in some way.

Reply:
How about 300ns?

Reply:
I'd do something much higher, we don't want to drop logs.
30s timeout should be fine.

Reply:
sounds good to me.